wolfssl/_build_ffi.py

```diff
@@ -391,6 +397,12 @@ def generate_libwolfssl():
     WOLFSSL_METHOD* wolfSSLv23_client_method(void);
 
     WOLFSSL_METHOD* wolfSSLv23_method(void);
+
+    WOLFSSL_METHOD* wolfDTLSv1_server_method(void);
+    WOLFSSL_METHOD* wolfDTLSv1_client_method(void);
+
+    WOLFSSL_METHOD* wolfDTLSv1_2_server_method(void);
+    WOLFSSL_METHOD* wolfDTLSv1_2_client_method(void);
```

Comment:
I believe we could add methods for DTLS 1.3 as well, by compiling wolfSSL with `--enable-dtls13` above.

Reply:
Done